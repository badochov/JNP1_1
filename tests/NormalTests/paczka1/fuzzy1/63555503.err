Error in line 10: oQa A628 4,5
