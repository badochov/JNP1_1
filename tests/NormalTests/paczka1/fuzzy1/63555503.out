979Q A 29,1
Agq84Nc9Wu S 111,1
KdLHswy A 108,0
wLL S 34,4
S60 34,4
S90 111,1
A615 108,0
A628 29,1
Agq84Nc9Wu A 179,3 S 111,1
wLL S 34,4
Agq84Nc9Wu A 179,3 S 111,1
